sonic-slave-buster/Dockerfile.j2

```diff
@@ -414,7 +419,7 @@ RUN apt-get update
 {%- if CONFIGURED_ARCH == "amd64" %}
 RUN apt-get install -y docker-ce=5:18.09.5~3-0~debian-buster docker-ce-cli=5:18.09.5~3-0~debian-buster
 {%- else %}
-RUN apt-get install -y docker-ce=18.06.3~ce~3-0~debian docker-ce-cli=18.06.3~ce~3-0~debian
+RUN apt-get install -y docker-ce=18.06.3~ce~3-0~debian
```

Comment:
why remove installing docker-ce-cli?

Reply:
docker was split into "engine", "cli", and "containerd" in 18.09.
versions before 18.09 integrated CLI in docker-ce.

refer to:
https://github.com/docker/docker-ce/releases/tag/v18.09.0
# Packaging
- Split "engine", "cli", and "containerd" to separate packages, and run containerd as a separate systemd service docker-ce-packaging#131, docker-ce-packaging#158

Reply:
what is reason that arm is not using the same docker engine? can arm use the same docker engine as x86?